cinder/README.md

```diff
@@ -0,0 +1,209 @@
+# Dell EMC Cinder driver containerization with OSP13
+
+## Overview
+
+Dell EMC Cinder driver integrates with the RedHat OpenStack Platform since OSP 6. In OSP13, all Cinder services are able to run within containers.
```

Comment:
It is not necessary to add this description. I would suggest to remove this line.

Reply:
Done